utils/fhirpath/fhir_path_processor.bal

```diff
@@ -103,6 +103,81 @@ public isolated function evaluateFhirPath(map<json> fhirResource, string fhirPat
 
 }
 
+# Select the resource elements from the given FHIR resource.
+#
+# + fhirPaths - FhirPath expressions for the required elements
+# + fhirResource - Requested fhir resource
+# + return - Returns FHIR resource with the selected elements
+public isolated function selectResourceElements(string[] fhirPaths, map<json> fhirResource) returns map<anydata>|error {
```

Comment:
This will accept a fhirpaths from different resources as well? asking this because you are accepting one FHIR resource at a time or are you expecting a bundle to be passed when the multiple fhir paths are from different resources?

Reply:
Atm, the logic is to not accept FHIR Path from different resources. `ResourceType` from the FHIR path will be checked before proceeding with the element extraction logic. So, the method is to select one or multiple FHIR Paths from a specific resource at a time only.